intg/src/main/python/apache_ranger/client/ranger_client.py

```diff
@@ -324,6 +324,11 @@ def __call_api(self, api, query_params=None, request_data=None):
             LOG.error("Ranger admin unavailable. HTTP Status: %s", HTTPStatus.SERVICE_UNAVAILABLE)
 
             ret = None
+        elif response.status_code == HTTPStatus.SERVICE_NOT_FOUND:
```

Comment:
Consider renaming SERVICE_NOT_FOUND => NOT_FOUND.

Reply:
thanks for the comments @mneethiraj, it makes sense